src/Components/Artwork/GridItem.tsx

```diff
@@ -57,11 +57,16 @@ class ArtworkGridItemContainer extends React.Component<Props, State> {
       return null
     }
 
+    let breakpoint
     if (this.state.isMounted) {
-      const [[match]] = Object.entries(breakpoints).filter(([key, val]) => val)
```

Comment:
i glossed over this in my code review yesterday (having never seen `Object.entries` before), assuming it something like `Object.keys`, except when `breakpoints: {}` (an empty object), like for gene/tag pages, this actually crashes!

So I just rewrote it in a less exciting but more predictable way.